app/app.rb

```diff
@@ -65,13 +71,16 @@ class DwellBNB < Sinatra::Base
 
   delete '/sessions' do
     session[:user_id] = nil
-    p current_user
     flash.keep[:notice] = 'Goodbye !'
     redirect to '/'
   end
 
+  delete '/book' do
```

Comment:
Please can this be changed its - I dont understand its purpose because it doesnt delete bookings so im 😕